sound/soc/sof/intel/hda-stream.c

```diff
@@ -297,7 +297,7 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 	struct hdac_stream *hstream = &stream->hstream;
 	int sd_offset = SOF_STREAM_SD_OFFSET(hstream);
 	int ret, timeout = HDA_DSP_STREAM_RESET_TIMEOUT;
-	u32 val, mask;
+	u32 val, mask, run;
```

Comment:
run is not set anywhere, have you actually tested this code?

Reply:
the snd_sof_dsp_read_poll_timeout is a #define defined by @kv2019i :
set by line:
(val) = snd_sof_dsp_read(sdev, bar, offset); \